pkg/metrics/metrics.go

```diff
@@ -333,11 +333,9 @@ var (
 
 	// Events
 
-	// EventTS*is the time in seconds since epoch that we last received an
-	// event that we will handle
-	// source is one of k8s, docker or apia
-
-	// EventTS is the timestamp of k8s resource events.
+	// EventTS is the time in seconds since epoch that we last received an
+	// event that was handled by Cilium. This metric tracks the source of the
```

Comment:
I don't really understand the “since epoch“ part. Isn't it the time since we last received the event? “seconds since epoch” is not a unit of time, let's just keep “seconds”, or am I missing something?

Reply:
> Isn't it the time since we last received the event

No, that would be a of absolute seconds since the event, this is a timestamp of when the event was received, so it's the number of seconds since the unix epoch.

Reply:
OK I get it, it's a timestamp and not a duration. I should have taken the clue from the `...TS` at the end of `EventTS`. Thanks for the clarification!